spec/openapi.yaml

```diff
@@ -306,13 +322,13 @@ paths:
         description: Unique ID of an individual listing
         required: true
         schema:
-          type: string
+          $ref: "#/components/schemas/ObjectID"
       - name: version_id
```

Comment:
Having `<listing_id>/<version_id>` here seems inconsistent with the `/listing_versions/<id>` scheme above. Also, if `version_id` is supposed to be globally unique, there wouldn't be a need to specify both ids here?

Reply:
The reason for splitting is to avoid having to search through multiple tables to know if something is a listing or a version. By having separate methods, it means that users can drill down into analytics based on a version or a listing without a slow down of determining type.

Reply:
As to consistency, this is a drilldown of a single type of data, where listing_versions support a new entity type in the data.